packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -537,21 +537,23 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 	}
 
 	/**
-	 * Apply changes from an op just as if a local client has made the change,
+	 * Apply changes from the provided op content just as if a local client has made the change,
 	 * including submitting the op. Used when rehydrating an attached container
-	 * with pending changes. This prepares the SharedObject for seeing an ACK
-	 * for the op or resubmitting the op upon reconnection.
-	 * @param content - Contents of a stashed op.
-	 * @returns Should return void.
+	 * with pending changes. The rehydration process replays all remote ops
+	 * and applies stashed ops after the remote op with a sequence number
+	 * that matches that of the stashed op is applied. This ensures
+	 * stashed ops are applied at the same state they were originally created.
+	 *
+	 * It is possible that stashed ops have been sent in the past, and will be found when
```

Comment:
Sometimes when I read documentation about stashed ops, it's easy to get confused with many terms (stashed - pending / remote - saved). Also, I would love an independent md file to understand more about the internal flow of applyStashedOp and not have to look at PR descriptions.

Reply:
This md file could have a glossary about all terms being used.